### PR TITLE
Suggest downloading root.hints as optional

### PR DESCRIPTION
**Pull request by richie510**
The root.hints file is automatically a dependency of unbound in the dns-root-data package, and would be updated automatically by the package manager.

### CODE REVIEW

File: docs/guides/unbound.md [hunk 1]
```diff
@@ -57,7 +57,9 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-**Important**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently.
+If you are installing unbound from a package manager, it should install the root.hints automatically with the dependency dns-root-data. The root hints will then be automatically updated by your packaged manager.
```
Comment (PromoFaux) [inline docs/guides/unbound.md:60]:
```suggestion
If you are installing unbound from a package manager, it should install the `root.hints` file automatically with the dependency `dns-root-data`. The root hints will then be automatically updated by your package manager. 
```

**Comment by dschaper** [comment]
docs/guides/unbound.md:60:212 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/guides/unbound.md:62:297 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]

File: docs/guides/unbound.md (hunk 1 above)
Comment (dschaper) [reply docs/guides/unbound.md:60]: Also add that the config file needs to have the line uncommented if the file is downloaded.

File: docs/guides/unbound.md [hunk 2]
```diff
@@ -57,7 +57,9 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-**Important**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently.
+If you are installing unbound from a package manager, it should install the root.hints automatically with the dependency dns-root-data. The root hints will then be automatically updated by your packaged manager. 
+
+**Optional**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently. This is only necessary if you are not installing unbound from a package manager.
```
Comment (PromoFaux) [inline docs/guides/unbound.md:62]:
```suggestion
**Optional**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently. This is only necessary if you are not installing unbound from a package manager.
```

Comment (dschaper) [inline docs/guides/unbound.md:62]: Add that the config file needs to be uncommented if the optional step is taken.

Comment (richie510) [reply docs/guides/unbound.md]: Done

**Comment by pralor** [comment]
This pull request has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/why-not-use-default-unbound-dependency-dns-root-data-for-root-hints-file/30252/21

File: docs/guides/unbound.md [hunk 3]
```diff
@@ -57,7 +57,9 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-**Important**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently.
+If you are installing unbound from a package manager, it should install the `root.hints` file automatically with the dependency `dns-root-data`. The root hints will then be automatically updated by your package manager.
+
+**Optional**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently. This is only necessary if you are not installing unbound from a package manager. If you do this optional step, you will need to uncomment the appropriate line in the suggested config file.
```
Comment (dschaper) [inline docs/guides/unbound.md:62]:
```suggestion
**Optional**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently. This is only necessary if you are not installing unbound from a package manager. If you do this optional step, you will need to uncomment the `root-hints:` configuration line in the suggested config file.
```

**Comment by dschaper** [comment]
That should do it for me.

**Comment by dschaper** [comment]
I think this is still under consideration on if it's required.

**Comment by XhmikosR** [comment]
@jfb-pihole @PromoFaux can you confirm these changes OK? I currently don't use unbound myself so I cannot verify.

**Comment by dschaper** [comment]
See the discussion at https://discourse.pi-hole.net/t/why-not-use-default-unbound-dependency-dns-root-data-for-root-hints-file/30252/21

**Review by DL6ER — APPROVED** [review]
Either way is okay with me. The root servers change rarely so even when the update is some time behind (because it has to go through the operating system repositories), there will still be plenty of other root servers be available in this time.

**Pull request merged**